sdks/python/apache_beam/runners/interactive/interactive_beam.py

```diff
@@ -86,10 +85,57 @@ def capture_duration(self, value):
       # The next PCollection evaluation will capture fresh data from sources,
       # and the data captured will be replayed until another eviction.
     """
+    assert value.total_seconds() > 0, 'Duration must be a positive value.'
     self.capture_control._capture_duration = value
 
   # TODO(BEAM-8335): add capture_size options when they are supported.
 
+  @property
+  def display_timestamp_format(self):
+    """The format in which timestamps are displayed.
+
+    Default is '%Y-%m-%d %H:%M:%S.%f%z', e.g. 2020-02-01 15:05:06.000015-08:00.
```

Comment:
Where is the default defined?

If defined somewhere else, let's not repeat these in comments. It will require sync both places going forward.

Reply:
How do we plan to keep the defaults in sync between here and interactive_options?